apps/cde-ui/src/app/pages/create-visualization-page/create-visualization-page.component.html

```diff
@@ -0,0 +1,120 @@
+<img class="img-bg" src="assets/background.png" />
+
+<div class="content">
+  <span class="title">Create a Cell Distance Visualization</span>
+
+  <mat-card class="creation-step">
+    <mat-card-header>
+      <mat-card-title>Format and Upload Data</mat-card-title>
+    </mat-card-header>
+    <mat-card-content> </mat-card-content>
+    <mat-card-actions>
+      <button mat-button *ngIf="!dataUploaded" (click)="upload()">Upload CSV</button>
```

Comment:
Use new control flow syntax for `if` blocks: https://angular.dev/guide/templates/control-flow